src/components/UNSTABLE__Shell/ShellToolbar/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file Toolbar.
+ * @copyright IBM Security 2020
+ */
+
+import { node } from 'prop-types';
+import React, { Children, cloneElement, useState } from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as shellNamespace } from '../Shell';
+import { namespace as toolbarNamespace } from '../../Toolbar/Toolbar';
+
+const namespace = appendComponentNamespace(shellNamespace, 'toolbar');
+
+function ShellToolbar({ children, ...other }) {
+  const [activeAction, setActiveAction] = useState(null);
+
+  return (
+    <nav className={`${toolbarNamespace} ${namespace}`} {...other}>
+      <div className={`${toolbarNamespace}__group`}>
+        {Children.map(children, (child, index) =>
+          cloneElement(child, {
```

Comment:
Can we avoid `cloneElement` here?

Reply:
Resolving thread in favor of https://github.com/carbon-design-system/ibm-security/pull/285#discussion_r384737204